docs/DeveloperGuide.md

```diff
@@ -121,11 +121,11 @@ The `Model`,
 
 The `Storage` component,
 * can save `UserPref` objects in json format and read it back.
-* can save the address book data in json format and read it back.
+* can save the patient book data in json format and read it back.
 
 ### Common classes
 
-Classes used by multiple components are in the `seedu.addressbook.commons` package.
```

Comment:
Thanks for pointing this out! I thought of this too but figured that it'll be better to change the package name in a separate PR as the current one only focusses on refactoring the AddressBook class. It will be easier for version control as well.